namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-court-register-dev/resources/variables.tf

```diff
@@ -11,7 +11,7 @@ variable "kubernetes_cluster" {
 variable "application" {
   description = "Name of the application you are deploying"
   type        = string
-  default     = "HMPPS Court Register"
+  default     = "hmpps_court_register"
```

Comment:
updating this will update other tags, see plan

Reply:
that's alright. I'm still working on getting the application to connect to resources (currently it doesn't)

Reply:
reverted the change